src/main/java/org/csanchez/jenkins/plugins/kubernetes/pod/yaml/YamlMergeStrategy.java

```diff
@@ -0,0 +1,16 @@
+package org.csanchez.jenkins.plugins.kubernetes.pod.yaml;
+
+import hudson.ExtensionPoint;
+import hudson.model.AbstractDescribableImpl;
+import io.fabric8.kubernetes.api.model.Pod;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+public abstract class YamlMergeStrategy extends AbstractDescribableImpl<YamlMergeStrategy> implements ExtensionPoint {
```

Comment:
You could just make this an `enum`, unless you expect there to be configurable fields. OTOH you then get an `Enum.name` rather than a possibly nicer `@Symbol`.